client/collection_view.coffee

```diff
@@ -43,7 +43,12 @@ Template._houston_collection_view.helpers
   custom_selector_error: -> Houston._session("custom_selector_error")
   field_filter_disabled: -> if Houston._session("custom_selector") then "disabled" else ""
   headers: -> get_collection_view_fields(@name)
-  nonid_headers: -> get_collection_view_fields(@name)[1..]
+  nonid_headers: ->
+    headers = get_collection_view_fields(@name)
+    if headers[0]?._required # do not remove _id if it is required by schema
+      headers
+    else
+      headers[1..]
```

Comment:
I'd like to rename it to edit_headers (and 'headers' to view_headers). Although it is very corner-case (and I don't use it myself) I can imagine that someone will add _id in schema with let's say type String (actually nothing else can be added I believe except useless in most applications logic ObjectId). 

Problem that it currently solves is disappearing _required_ field _id from create screen rendering creating impossible if there's required _id in schema.